content/rooms/fragments/v11-redactions.md

```diff
@@ -0,0 +1,38 @@
+---
+---
+
+{{% added-in this=true %}} The top-level `origin`, `membership`, and `prev_state` properties
+are no longer protected from redaction. The `m.room.create` event no longer keeps the `creator`
+property and now keeps the entire `content` property. The `m.room.redaction` event keeps the
+`redacts` property under `content`. The `m.room.power_levels` event keeps the `invite` property
+under `content`.
```

Comment:
The reviewer should go through this with a fine-tooth comb compared to the MSC3820's sub-MSCs. I think I got them all (and reflected the changes properly below), but it is easy to make a mistake.

Reply:
lgtm